src/components/CustomAppBar/CustomAppBar.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Toolbar } from '@mui/material';
+
+import { Logo } from 'components';
+
+import { StyledAppBar } from './styles';
+
+export const CustomAppBar = () => {
+  return (
+    <StyledAppBar>
+      <Logo />
+      <Toolbar disableGutters></Toolbar>
+    </StyledAppBar>
+  );
+};
```

Comment:
if there is no logics, you could remove return statement
```suggestion
const CustomAppBar = () =>  (
    <StyledAppBar>
      <Logo />
      <Toolbar disableGutters></Toolbar>
    </StyledAppBar>
  );
```